ACCESSIBILITY.md

```diff
@@ -0,0 +1,100 @@
+# Accessibility Policy of The Turing Way Community
+
+Within _The Turing Way_ community, we define accessibility practices as a set of community-wide, behavioral, social, and technical decisions that can be taken to ensure that all are welcome and are able to enter and participate in research, data-driven processes, communities and events despite any barriers for access.
+
+While accessibility is commonly understood with respect to disability, we – meaning _The Turing Way_ community – define accessibility as giving access to everyone, with specific attention given to ensuring proper access for groups of people that encounter different barriers for participation. Barriers may be anything related to, including, and extending beyond the following: 
+- Age
+- Disability
+- Ethnicity and race
+- Family
+- Language (written and/or spoken)
+- Geography
+- Nationality
+- Religion, faith and belief
+- Sex and gender
+- Sexual orientation
+- Socioeconomic status
+- Technical or disciplinary background
+
```

Comment:
Adding something related to design here after discussing with @Preshh0 and @LizHareDogs about Design (#3930)